python/pyspark/rdd.py

```diff
@@ -181,6 +181,7 @@ def __init__(self, jrdd, ctx, jrdd_deserializer=AutoBatchedSerializer(PickleSeri
         self._jrdd = jrdd
         self.is_cached = False
         self.is_checkpointed = False
+        self.is_locally_checkpointed = False
```

Comment:
No it's not. I just mirrored the behavior of self.is_checkpointed: the flag self.is_locally_checkpointed is set by the Python wrappers, but isLocallyChecked() wraps around the scala equivalent, which I imagine reads itself a scala flag.